### PR TITLE
Make edit_rest add a \n at end of file

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Following the changes in #67 and #68

### CODE REVIEW

**Comment by Daniel-at-github** [comment]
@zerok Done. 
I have doubts with two values more, I added they  commented

File: .editorconfig [hunk 1]
```diff
@@ -3,3 +3,6 @@ root = true
 [*.json]
 indent_size = 2
 indent_style = spaces
+insert_final_newline = true
+# end_of_line = lf
```
Comment (zerok) [inline .editorconfig]: Both should be fine :) I'd even go as far as moving insert_final_newline, end_of_line and charset to [*].

**Comment by zerok** [comment]
Sorry, first time I'm using the new code-review feature on Github. The "red" color is a bit strong here ;)

**Comment by Daniel-at-github** [comment]
Changes done.

I tried to https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/ but no `Allow edits from maintainers` found :confused:

**Comment by zerok** [comment]
Thanks :)

**Pull request merged**